doc/man1/ts.pod

```diff
@@ -216,9 +216,10 @@ otherwise it is a time stamp token (ContentInfo).
 
 =item B<-config> configfile
 
-The configuration file to use, this option overrides the
-B<OPENSSL_CONF> environment variable. See B<CONFIGURATION FILE
-OPTIONS> for configurable variables. (Optional)
+The configuration file to use.
+Optional; for a description of the default value,
+see L<openssl(1)/COMMAND SUMMARY>.
+See B<CONFIGURATION FILE OPTIONS> for configurable variables.
```

Comment:
well, it's defined in two separate sections for each of -query -reply.
I think refactoring out common flags is a good thing to do, but a separate PR.